tests/integration/operator/env_utils.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integrationoperator
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+
+	g "github.com/onsi/ginkgo/v2"
+)
+
+var (
+	ocp                   = getBoolEnvOrDefault("OCP", false)
+	skipDeploy            = getBoolEnvOrDefault("SKIP_DEPLOY", false)
+	image                 = getEnvOrDefault("IMAGE", "quay.io/maistra-dev/istio-operator:latest")
```

Comment:
nit: I'd simplify this to just `getEnv`. It's clear that the second argument is the default value, so there's no need to say "OrDefault" in the function name.

Same for `getBoolEnvOrDefault`.

Reply:
Done